tools/driver/packages/packages.go

```diff
@@ -137,15 +147,15 @@ func Load(req *DriverRequest, files []string) (*DriverResponse, error) {
 			WordSize: 8,
 			MaxAlign: 8,
 		},
-		Packages: append(pkgs, stdlib...),
+		Packages: pkgs,
 		Roots:    roots,
 	}, nil
 }
 
 // loadPackageInfo loads all the package information by executing Please.
 // A cooler way of handling this in future would be to do this in-process; for that we'd
 // need to define the SDK we keep talking about as a supported programmatic interface.
-func loadPackageInfo(files []string) ([]*packages.Package, error) {
+func loadPackageInfo(files []string, needExportFile bool) ([]*packages.Package, error) {
```

Comment:
I might be missing it but `needExportFile` doesn't look like it gets used in `loadPackageInfo`? Also I'm seeing the ExportFile path missing from the packages in the json response... is `pkg.ExportFile` guaranteed to be populated?
```"Packages": [
    {
        "ID": "common/go/db/dberrors/dberrclasses",
        "Name": "dberrclasses",
        "PkgPath": "common/go/db/dberrors/dberrclasses",
        "GoFiles": [
            "/home/swestmoreland/core3/src/common/go/db/dberrors/dberrclasses/classes.go"
        ],
        "CompiledGoFiles": [
            "/home/swestmoreland/core3/src/common/go/db/dberrors/dberrclasses/classes.go"
        ],
        "ExportFile": "/home/swestmoreland/core3/src/plz-out/gen"
    },
    {
        "ID": "common/go/registry",
        "Name": "registry",
        "PkgPath": "common/go/registry",
        "GoFiles": [
            "/home/swestmoreland/core3/src/common/go/registry/formatter.go",
            "/home/swestmoreland/core3/src/common/go/registry/library.go",
            "/home/swestmoreland/core3/src/common/go/registry/registry.go",
            "/home/swestmoreland/core3/src/common/go/registry/seq.go"
        ],
        "CompiledGoFiles": [
            "/home/swestmoreland/core3/src/common/go/registry/formatter.go",
            "/home/swestmoreland/core3/src/common/go/registry/library.go",
            "/home/swestmoreland/core3/src/common/go/registry/registry.go",
            "/home/swestmoreland/core3/src/common/go/registry/seq.go"
        ],
        "ExportFile": "/home/swestmoreland/core3/src/plz-out/gen",
        "Imports": {
            "fmt": "fmt",
            "sync": "sync"
        }
    },
```

Reply:
Sorry ignore me! Was using the old version of the plugin -.-

Reply:
You're right that the argument wasn't used though!